server/server.go

```diff
@@ -58,6 +58,7 @@ func (a *Server) initRoutes() {
 	r.Use(middleware.Recoverer)
 	r.Use(middleware.Compress(5, "application/xml", "application/json", "application/javascript"))
 	r.Use(middleware.Heartbeat("/ping"))
+	r.Use(requestLogger)
```

Comment:
This lines needs to be added _after_ the `injectLogger` one, or else the log message won't include the requestId.

Also, as I said in my [comment in the issue](https://github.com/navidrome/navidrome/issues/836#issuecomment-803683746), we need to remove the `requestLogger` from the `MountRouter` function, it is unnecessary there and it could potentially cause double messages.

Reply:
Oops, I misread you comment. Should be ok now, sorry.